doc/integrator/cluster.rst

```diff
@@ -0,0 +1,23 @@
+.. _integrator_cluster:
+
+Create a cluster
+================
+
```

Comment:
Something like:
A cluster is when we want to use more than one server to serve the application
?